### PR TITLE
Basic Rules conformity to the style guide and other

### PR DESCRIPTION
**Pull request by 1sourcecontrol**
- Edited out colons in some chapter titles... I think it looks cleaner.
- Made styles more consistent by increasing # of "=" signs in section declarations.
- Also, scope creep, I noticed that a definition for Nanites never got established so I added some text. (I know there is some flux about this, but let's put something there until further notice).
- Fixed some text in the cloaking section
- Reworded some text in combat stat examples

### CODE REVIEW

File: Basic Rules.txt [hunk 1]
```diff
@@ -117,9 +115,14 @@ Class 		- Think of this like your job. Are you a sniper, waiting for the
...
-	money you start with to buy equipment and how many nanites you can use
+	money you start with to buy equipment and how many Nanites you can use
 	at once.
-Nanites  	- 
+Nanites  	- The evolution of technology brought about the ability to design
+	and control microscopic machines. Such a powerful advancement assists in
+	all forms of life in the Compound X universe. Nanites are used to activate
+	abilities that are not normally possible on Earth. Abilities such as sprinting 
+	extremely fast, aiming incredibly well, infusing bullets with electrical
+	energy, and more.
```
Comment (bleehu) [inline Basic Rules.txt]: While I'm normally quite loathe to accept scope creep, I think you handled it pretty well here. I do think "not normally possible _on earth_ " is a little non-diajetic and I'd like to see if we can tweak it, but that's a nitpick. Great examples!

**Review by bleehu — APPROVED** [review]
shipit!

File: Basic Rules.txt (hunk 1 above)
Comment (1sourcecontrol) [reply Basic Rules.txt]: I'll try and change it to something more in-game. I noticed that other areas seemed to break some of the '4th wall' if you will, so I was trying to build some excitement by comparing what's possible on earth.

**Comment by bleehu** [comment]
Looks good! Shipit! Great work, 1Source!
